scripts/parse.js

```diff
@@ -5,8 +5,8 @@ import crypto from "crypto";
 
 const MODULES_DIR = path.join("modules");
 const OUTPUT_FILE = path.join(config.dist.directory, "modules.json");
-const ACCEPTED_EXTENSIONS = [".json", ".js", ".ts", ".jsx", ".tsx", ".md", ".py"];
-const META_FILE = ["meta.json"]
+const ACCEPTED_EXTENSIONS = [".json", ".js", ".ts", ".jsx", ".tsx", ".md", ".py", ".png"];
```

Comment:
```suggestion
const ACCEPTED_EXTENSIONS = [".json", ".js", ".ts", ".jsx", ".tsx", ".md", ".py"];
const PREVIEW_IMAGE = ["preview.png"];
```

Reply:
Thank you. I didn't think about the fact that we could accidentally parse more than just the preview so this change makes sense!